src/commitizen/commit.js

```diff
@@ -1,5 +1,5 @@
 import path from 'path';
-import homeOrTmp from 'home-or-tmp';
+import ConfigDir from 'application-config-path';
```

Comment:
I think that the convention is that initial uppercase letter is for classes, can you change this name to `configDir` instead? thanks